samples/azure/azure-functions/AzureFunctions_1/AzureFunctions.ASBTrigger/AzureFunctions.ASBTrigger.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <AzureFunctionsVersion>v2</AzureFunctionsVersion>
+    <LangVersion>7.1</LangVersion>
```

Comment:
do we have a special guideline at what point we're explicitly specifying language version and when not?